Dockerfile

```diff
@@ -24,8 +24,6 @@ RUN apt-get update && \
     apt-get install -y vim-tiny && \
     pip install --upgrade pip
 
-RUN pip install -e git://github.com/reanahub/reana-commons.git@master#egg=reana-commons
```

Comment:
Missing `reana-commons`. You can add it just before `COPY . /code`, also `reana-workflow-commons` could be moved before.

Reply:
Adding it, in https://github.com/reanahub/reana-workflow-engine-cwl/pull/46 @tiborsimko suggested adding it after the `COPY . /code`, I am not sure if it affects the rebuilding though.

Reply:
Because until we don't release `reana-commons` and `reana-workflow-commons` in PyPI we will have to rebuild the image with no cache if we change any of the repos since the docker builder will take it as cached. You can move them after and once we release on PyPI we remove the lines, I just made the comment because it is pulling from Github the two repos and installing them every time I rebuild, therefore slowing down the rebuild.

Lets follow @tiborsimko advise then:
> I would propose the safest order like:
pypi requirements builder all to cache
COPY . /code
pip r-commons r-w-commons
pip install .

Reply:
I was referring to the order `1. COPY 2. pip install -e git://...` if it affected the docker image building, the rest is of course obvious. The order is already as described